airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -20,26 +20,39 @@
 """
 This module contains Google BigQuery operators.
 """
+import datetime
 import enum
 import hashlib
 import json
+import logging
+import random
 import re
+import string
 import uuid
 import warnings
+import uuid
 from typing import Any, Dict, Iterable, List, Optional, Sequence, Set, SupportsAbs, Union
+from urllib.parse import urlsplit
 
 import attr
 from google.api_core.exceptions import Conflict
 from google.cloud.bigquery import TableReference
 
+from airflow.providers.greatexpectations.operators.greatexpectations_base import GreatExpectationsBaseOperator
+from great_expectations.data_context.types.base import DataContextConfig
+from great_expectations.data_context import BaseDataContext
```

Comment:
I'm not sure about it. Having GE should not be required for BigQuery operators. I think we can treat GE as separate provider and keep everything related to it there - including BigQuery specific operators. WDYT @mik-laj @kaxil @feluelle ?

Reply:
+1 for moving all `great_expectations` imports to its provider package.